### PR TITLE
feat(questions): add image input question type

### PR DESCRIPTION
**Pull request by pabloelisseo**
### Type:

- [ ] CI/CD: helm, docker & CI/CD adjustments.
- [x] feature: new capabilities.
- [ ] fix: bug, hotfix, etc.
- [ ] refactor: enhancements.
- [ ] style: changes in styles.
- [ ] other: docs, tests.

### Before submitting this PR, I made sure:

- [ ] There is no lint error in the code
- [ ] Build process passes successfully
- [ ] There are some tests

### CODE REVIEW

**Review by NuriaExtremadouro — APPROVED** [review]
Will it be stylable enough? Otherwise LGTM!

File: src/builder.js [hunk 1]
```diff
@@ -269,6 +273,13 @@ const FormBuilder = ({
       if (data[key] instanceof Date) {
         data[key] = data[key].toISOString()
       }
+      if (data[key] instanceof FileList) {
+        const reader = new FileReader()
+        reader.onload = function (e) {
```
Comment (juanma-dominguez) [inline src/builder.js:278]: should we include here an  reader.onerror callback?

**Review by juanma-dominguez — APPROVED** [review]
just a question

File: src/builder.js (hunk 1 above)
Comment (pabloelisseo) [reply src/builder.js:278]: Good spot! I have also realised that this should be asynchronous. Uploading fix!

**Comment by pabloelisseo** [comment]
> Will it be stylable enough? Otherwise LGTM!

It will be stylable the same way as normal inputs for now. I think it should be enough, at least for DAZN. Later when we add the image previewer we may need to create a varian just for the image_input type

**Review by juanma-dominguez — APPROVED** [review]
LGTM

**Review by NuriaExtremadouro — APPROVED** [review]

**Review by elizabethLomb — APPROVED** [review]

**Pull request merged**